smoke_test.sh

```diff
@@ -0,0 +1,63 @@
+# Spin up iqe pod and execute IQE tests in it
+
+# Env vars defined by caller:
+#IQE_PLUGINS="plugin1,plugin2" -- pytest plugins to run separated by ","
+#IQE_MARKER_EXPRESSION="mymarker" -- pytest marker expression
+#IQE_FILTER_EXPRESSION="something AND something_else" -- pytest filter, can be "" if no filter desired
+#NAMESPACE="mynamespace" -- namespace to deploy iqe pod into, can be set by 'deploy_ephemeral_env.sh'
+
+#IQE_POD_NAME="iqe-tests"
+
+# create a custom svc acct for the iqe pod to run with that has elevated permissions
+# SA=$(oc get -n $NAMESPACE sa iqe --ignore-not-found -o jsonpath='{.metadata.name}')
+# if [ -z "$SA" ]; then
+#     oc create -n $NAMESPACE sa iqe
+# fi
+# oc policy -n $NAMESPACE add-role-to-user edit system:serviceaccount:$NAMESPACE:iqe
+# oc secrets -n $NAMESPACE link iqe quay-cloudservices-pull --for=pull,mount
+oc apply -n $NAMESPACE -f $APP_ROOT/deploy/rbac-cji-smoketest.yml
+
+job_name=rbac-smoke-tests-iqe
+found=false
+end=$((SECONDS+45))
+
+echo "Waiting for Job $job_name to appear"
+
+while [ $SECONDS -lt $end ]; do
+    if `oc get job $job_name -n $NAMESPACE >/dev/null 2>&1`; then
+        found=true
+        break
+    fi
+    sleep 1
+done
+
+if [$found == "false"] ; then
+    echo "Job $job_name failed to appear"
+    exit 1
+fi
+
+echo "Waiting for Job $job_name to be running"
+running=false
+pod=""
+
+while [ $SECONDS -lt $end ]; do
+    pod=$(oc get pods -n $NAMESPACE -o json | jq -r '.items[] | select(.status.phase=="Running") | select(.metadata.name|test("rbac-smoke.")) .metadata.name')
+    if [ -z $pod ]; then
+        running=true
+        break
+    fi
+    sleep 1
+done
+
+if [$running == "false"] ; then
```

Comment:
In this case I think you'd need to insert some space before `$running` and maybe wrap it in quotes for good measure. lmk if you want me to keep working in this branch and i'll tack on some commits.

Reply:
Thanks! My bash is so bad :man_facepalming:

Reply:
Nah, it's fine. I've accidentally done this plenty of times :)